R/status.R

```diff
@@ -58,7 +111,7 @@ renv_status_impl <- function(project, libpaths, lockpath, sources, cache) {
 
   # check to see if we've initialized this project
   if (!renv_project_initialized(project)) {
-    writef("* This project has not yet been initialized.")
+    writef("This project has not yet been initialized.")
```

Comment:
I wonder if these should be errors.

Reply:
I don't think so; IMHO `status()` shouldn't ever give an error, it should just give advice for getting the project into a "good" state.

Reply:
My reasoning is that it might make sense for it to error when called on something that's not an renv project; you can obviously fix that by calling `renv::init()` but it might be more likely to reflect a genuine error? i.e. you're in the wrong directory.

Reply:
Ah, that's a good point...

What if the user (1) hasn't loaded renv but (2) the working directory is in an renv project? (What if it's a sub-directory of an renv project?)

Reply:
What if we error if there's neither a project library nor a lockfile, but otherwise give a helpful message (i.e. call `snapshot()` if no lockfile; call `restore()` if no library)?

Reply:
I think that makes sense.

Reply:
I've left these all as messages, not errors, but tweaked them all to be more immediately actionable.